docs/src/man/doctests.md

```diff
@@ -191,6 +191,35 @@ help avoid unintentional definitions in following doctest blocks.
     The `DocTestSetup` value is **re-evaluated** at the start of *each* doctest block
     and no state is shared between any code blocks.
 
+## Filtering Doctests
+
+A part of the output of a doctest might be non-deterministic, e.g. pointer addresses and timings.
+It is therefore possible to filter a doctest so that the deterministic part can still be tested.
+A filter takes the form of a regular expression.
+Each match in the expected output and the actual output is removed before the outputs are compared. 
+Filters are added globally i.e. applied to all doctests in the documentation by giving a list of regular expressions to
```

Comment:
... and this one too. Splitting this paragraph up a bit would improve readability a bit in my opinion.